Examples.fshx

```diff
@@ -38,11 +50,79 @@ InstanceOf: CancerPatient
 * communication.language = urn:ietf:bcp:47#en-US "English (Region=United States)"
 * communication.language.text = "English"
 
+
+Instance: mCODEPatientExample02
+InstanceOf: CancerPatient
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/CancerPatient"
+* extension[us-core-race].extension[ombCategory].valueCoding = urn:oid:2.16.840.1.113883.6.238#2186-5 "Not Hispanic or Latino"
```

Comment:
I am not sure if you can refer to the us-core-race extension like this.  You might have to do:
Alias: USCoreRace = http://hl7.org/fhir/us/core/StructureDefinition/us-core-race
...
* extension[USCoreRace].extension[ombCategory].valueCoding = ...

but I'm not sure. It's all new!